app/helpers/permission-to-remove-contributor.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+
+/**
+* permissionToRemoveContributor helper.  Checks to see if user has proper permissions
+* to remove contributor.  The user either must be an admin or trying to remove herself.
+* The project cannot be a registration.
+*/
+export function permissionToRemoveContributor(params/*, hash*/) {
+    var contributor = params[0];
+    var currentUser = params[1];
+    var stillAdmin = params[2];
+    var node = params[3];
```

Comment:
Not important and tangential syntax note:

[Destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) is a nice concise way to do this in ES6.